.changeset/curious-mountain-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fluidframework/merge-tree": major
+"@fluidframework/sequence": major
+---
```

Comment:
```suggestion
---
"@fluidframework/merge-tree": minor
"@fluidframework/sequence": minor
---
```

You didn't make any breaking changes here (which is correct, since it's going into main). "minor" more accurately communicates that this introduces a new feature for interval collections.